flask_appbuilder/hooks.py

```diff
@@ -0,0 +1,76 @@
+def before_request(*args, **kwargs):
```

Comment:
FAB is not totally type annotated, but every new addition should be.

Reply:
👍🏻